components/infobox/wikis/infobox_league_custom.lua

```diff
@@ -0,0 +1,113 @@
+---
+-- @Liquipedia
+-- wiki=pokemon
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Logic = require('Module:Logic')
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _GAME = mw.loadData('Module:GameVersion')
+local _MODES = mw.loadData('Module:GameModes')
+local _FORMATS = mw.loadData('Module:GameFormats')
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+	_args.format = CustomLeague:_getGameFormat()
+
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.liquipediaTierHighlighted = CustomLeague.liquipediaTierHighlighted
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'gamesettings' then
+		return {
+			Cell{name = 'Game version', content = {
+					CustomLeague._getGameVersion()
+				}
+			},
+			Cell{name = 'Game mode', content = {
+					CustomLeague:_getGameMode()
+				}
+			},
+		}
+	elseif id == 'customcontent' then
+		if _args.player_number then
+			table.insert(widgets, Title{name = 'Players'})
+			table.insert(widgets, Cell{name = 'Number of players', content = {_args.player_number}})
+		elseif _args.team_number then
+			table.insert(widgets, Title{name = 'Teams'})
+			table.insert(widgets, Cell{name = 'Number of teams', content = {_args.team_number}})
+		end
+	end
+	return widgets
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.game = args.game
```

Comment:
Isn't it better to use the normalized game name instead of the raw input? (i.e CustomLeague._getGameVersion())

Reply:
yeah, normalized should be the way (since **|game=** were always an input that matches the **Module:GameVersion** calls). What should be the code changes here

(replaced args.game with just **CustomLeague._getGameVersion()** ?)